Makefile

```diff
@@ -706,6 +706,7 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             telemetry/esc_telemetry.c \
 
 SIZE_OPTIMISED_SRC := $(SIZE_OPTIMISED_SRC) \
+            fc/fc_msp.c \
```

Comment:
Right, I did not think about this. Do you have any other proposal? Someone suggested that removing GPS from BETAFLIGHTF3 would be alright as well.